internal/cleaner/processor.go

```diff
@@ -22,15 +22,17 @@ func ProcessNamespaces(
 	graph *msgraphsdk.GraphServiceClient,
 	kube kubernetes.Interface,
 	cfg *config.Config,
+	now time.Time,
```

Comment:
Would it be better to just call `time.Now()` in this function, instead of passing it as an input for something that just comes from an import?
Or was this designed like this to help with testing? If so, is there a more accurate name for this input than `now`?

Reply:
I updated the code to use the parameter name "referenceTime".